plugin/src/main/java/io/jenkins/plugins/casc/Attribute.java

```diff
@@ -270,14 +263,13 @@ public CNode describe(Owner instance, ConfigurationContext context) throws Confi
         }
     }
 
-
     /**
      * This function is for the JSONSchemaGeneration
      * @param instance Owner Instance
      * @param context Context to be passed
      * @return CNode object describing the structure of the node
      */
-    public CNode describeStructure(Owner instance, ConfigurationContext context) {
+    public CNode describeForSchema(Owner instance, ConfigurationContext context) {
```

Comment:
DescribeStructure would then no longer be needed as the describeForSchema would essentially perform  the check.

Reply:
Do we still need the mode var?

Reply:
We would not need it in the `describeForSchema` anymore because if `isJsonSchema` is set then it would anyways be calling `describeStructure` which would in turn call `describeForSchema`

Reply:
ok please remove then

Reply:
Done:+1: